docs/build-n-test.md

```diff
@@ -23,6 +23,8 @@
     - ``BUILD_STAT``: show the statistics of the PoW information.
     - ``BUILD_DEBUG``: dump verbose messages internally.
                        Build with the corresponding Sanitizer to detect software potential bugs if the value is `address`, `undefined` or `thread`.
+    - ``BUILD_BOARD``: specify the board which builds the source code. The supported boards are `de10nano` (DE10-nano board), `arrowsockit` (Arrow SoCKit board)
+                       and `rpi2/3` (Raspberry Pi 2/3 board).
```

Comment:
How about changing it to `BOARD`? `BUILD_BOARD` sounds weird.

Reply:
Also, the value specifying Raspberry Pi 2/3 can be `rpi` since only few people are using Raspberry Pi generation 1.

Reply:
I will change it to `BOARD`.

---

The available values are `rpi2` and `rpi3` rather than `rpi2/3`.
Their compiler flags are slightly different.
I'll change the description of the document and commit message to avoid confusion.

Reply:
Done.